install.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
```

Comment:
While having install.sh is a good thing overall, it implicitly assumes that the users who are going to try KubePlus are on a Linux machine. We have seen previously where users were on windows machine (including WSL). For such users, having direct kubectl based steps like we currently have are more helpful. 

So I suggest we keep both. We don't have to remove the current installation instructions from the README. We keep them and we can also have this install.sh for advanced users.

Reply:
Updated the README to inculcate the changes.